### ISSUE TITLE
Add a toy weather-to-load model

### ISSUE THREAD

**Issue by williamhobbs** [williamhobbs/hefty#15]
Start with a simple piece-wise linear model based either only on temperature, or combined temperature and wet bulb. Minimum of 2 or 3 segments (cooling, heating, maybe deadband).

Later versions could include more weather variables, plus time-based variables (weekday vs weekend, hour of day, sine/cosine of day of year, solar position, etc.)

See the following for ideas:
- https://docs.nlr.gov/docs/fy12osti/54509.pdf
- https://docs.nlr.gov/docs/fy15osti/64297.pdf
- https://www.nyiso.com/documents/20142/20053386/LFU_Whitepaper_Phase1_Results.pdf (two ways to combine temperature and humidity, based on dry bulb and wet bulb)

**Comment by williamhobbs** [comment]
Some good stuff by Dan Kirk-Davidoff at EPRI: https://www.epri.com/research/sectors/technology/results/3002031482.

Includes things like slope of heating and cooling loads with temp, linear vs XGBoost models, GHI dependence (GHI typically reduces load for the systems analyzed, presumably due to BTM solar).

**Comment by williamhobbs** [comment]
From a conversation with Dan:
- Consider a model that estimates a delta in load (e.g., based on the previous day or few days) relative to a delta in weather from that same time period. A "first derivative" model (my term)
- He saw promise in this a while ago, but was not able to fully explore it
- might make sense to do with something like XGBoost (and could use other logical features like day of week, etc.)

Some of my thoughts:
- this could help remove some of the challenges around normalizing historical training data
  - load delta in MW vs load delta as a percentage could have some differences here. Making it a percentage (i.e., change in load normalized to the previous load) might help 
- might reduce likelihood of large unrealistic steps changes in load
